server/src/root-model.ts

```diff
@@ -73,6 +73,11 @@ export class RootModel {
 	}
 	public addPlayer(player: PlayerModel) {
 		this.players[player.id] = player
+		const time = Date.now()
+		const date = new Date(time)
+		console.log(
+			`${date.toLocaleTimeString('it-IT')}: Player [${player.uuid}] has logged in.`,
+		)
```

Comment:
I'm not sure if this is the best idea for console readability, although we're long past the times when we used to check it to see how many players were online